src/test/java/com/rultor/agents/github/CommentsTagTest.java

```diff
@@ -93,6 +99,86 @@ public void duplicatesRelease() throws Exception {
     }
 
     /**
+     * CommentsTag cannot release an older version.
+     * @throws IOException In case of error.
+     */
+    @Test
+    public void rejectsOldRelease() throws IOException {
+        final Repo repo = new MkGithub().randomRepo();
+        final Issue issue = repo.issues().create("", "");
+        final Agent agent = new CommentsTag(repo.github());
+        repo.releases().create("1.0");
+        repo.releases().create("2.0");
+        repo.releases().create("3.0-b");
+        final Talk talk = CommentsTagTest.talk(issue, "1.5");
+        agent.execute(talk);
+        final Comment.Smart response = new Comment.Smart(
+            repo.issues().get(1).comments().get(1)
+        );
+        MatcherAssert.assertThat(
+            response.body(),
+            Matchers.containsString("version tag is too low")
+        );
+    }
+
+    /**
+     * CommentsTag can validate or reject a version.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void canValidateVersion() throws Exception {
+        Assert.assertTrue(
+            "Version 5.0 should be valid", CommentsTagTest.invokeValid("5.0")
+        );
+        Assert.assertTrue(
+            "Version 0.1 should be valid", CommentsTagTest.invokeValid("0.1")
+        );
+        Assert.assertTrue(
+            "Version .1.2.3.4 should be valid",
+            CommentsTagTest.invokeValid(".1.2.3.4")
+        );
+        Assert.assertFalse(
+            "Version beta should be invalid",
+            CommentsTagTest.invokeValid("beta")
+        );
+        Assert.assertFalse(
+            "Version 1.0-alpha should be invalid",
+            CommentsTagTest.invokeValid("1.0-alpha")
+        );
+        Assert.assertFalse(
+            "Version 1. should be invalid", CommentsTagTest.invokeValid("1.")
+        );
+        Assert.assertFalse(
+            "Version a.b.c should be invalid",
+            CommentsTagTest.invokeValid("a.b.c")
+        );
+    }
+
+    /**
+     * CommentsTag can validate a release.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void canValidateRelease() throws Exception {
+        final Collection<DefaultArtifactVersion> previous = new ArrayList<>(3);
```

Comment:
@gumbelmj Let's get rid of this test. `rejectsOldRelease` already validate behaviour if release is old.